test_util/integration_test.py

```diff
@@ -1361,6 +1363,14 @@ def wait_for_endpoint():
         'mesos-dns-service',
         'mesos-master-service',
         'signal-service']
+
+    ee_master_units = [
```

Comment:
This should not be in open. Variants should have a clean separation and not bleed through like this.